package.json

```
@@ -14,7 +14,7 @@
     "typecheck": "tsc",
     "build": "yarn clean && yarn rollup -c",
     "clean": "rimraf dist",
-    "ci": "yarn lint && yarn test && yarn test:versions && yarn test:types && yarn typecheck && yarn build",
+    "ci": "yarn lint && yarn test && yarn test:versions && yarn test:types && yarn typecheck",
```

nit: Is `ci` still the appropriate description? Wondering if it shouldn't be something like

```suggestion
    "test": "yarn test:lint && yarn test:unit && yarn test:versions && yarn test:types && yarn test:typecheck",
```